### ISSUE TITLE
Crashes after start

### ISSUE THREAD

**Issue by tersor** [jpegjames/glukey#6]
The application starts as normal after downloading, but the settings window is behind other windows (as mentioned in #5).

But after saving options, the app crashes. It continue to crash seconds after trying to start again. Details in [this crash report](https://gist.github.com/tersor/10389022f3120edcdca87ae16fb348cd).

I used my Dexcom username in the username field (and not email address).

**Comment by jpegjames** [comment]
Hey @tersor,

Sorry for the delay—this was posted right at the beginning of a long vacation where I was out of pocket for a while. I believe I have found the bug. It crashes when there is more than 24 hours of no data. I have fixed it on my end and want to track #5 before resubmitting to the App Store (hopefully later this week). 

I will close this one once it is live on the App Store and let you know. 

Thanks,
James
